UnicornRender/utility/sources/asset/Storage.cpp

```diff
@@ -26,7 +26,7 @@ void Storage::InitializeWorkers(uint16_t count)
         StopWorkers();
     }
 
-    LOG_INFO("Initializing %u workers", count);
+    spdlog::get(utility::Logger::LoggerName())->info("Initializing %u workers", count);
```

Comment:
`std::shared_ptr<spdlog::logger>` should be a class variable. There is no need to call `get()` every time.